tests/visual/select.css

```diff
@@ -0,0 +1,377 @@
+* {
```

Comment:
It will return us many redundant files. Why we don't compile css in vellum and link it to our components if we need it?

![screenshot 2015-03-13 15 07 18](https://cloud.githubusercontent.com/assets/4110509/6648045/6416e72e-c992-11e4-838f-b379997d208a.png)